src/cli.rs

```diff
@@ -173,6 +175,20 @@ A backslash can be used to escape special characters (e.g. weird\:key=value).
     /// Optional key-value pairs to be included in the request.
     #[structopt(skip)]
     pub request_items: Vec<RequestItem>,
+
+    /// Set to "no" (or "false") to skip checking the host's SSL certificate.
+    /// Defaults to "yes" ("true"). You can also pass the path to a CA bundle
+    /// file for private certs.
+    #[structopt(long, default_value, name = "VERIFY")]
+    pub verify: Verify,
+
+    /// Use a client side certificate for SSL.
+    #[structopt(long, name = "CERT")]
+    pub cert: Option<PathBuf>,
+
+    /// Pass the path to a private key file if the private key is not contained in the cert file.
+    #[structopt(long = "cert-key", name = "CERT_KEY")]
+    pub cert_key: Option<PathBuf>,
```

Comment:
This would make the help easier to skim by pointing out the type of the argument:
```suggestion
    /// Use a client side certificate for SSL.
    #[structopt(long, name = "FILE")]
    pub cert: Option<PathBuf>,

    /// Pass the path to a private key file if the private key is not contained in the cert file.
    #[structopt(long = "cert-key", name = "FILE")]
    pub cert_key: Option<PathBuf>,
```

Reply:
My bad, turns out they aren't allowed to have the same name like that :grimacing:

If you use `value_name =` instead of `name =` it works.

Reply:
`value_name` works!